### ISSUE TITLE
Question about text_scale

### ISSUE THREAD

**Issue by Pay20Y** [argman/EAST#244]
In model.py 
geo_map = slim.conv2d(g[3], 4, 1, activation_fn=tf.nn.sigmoid, normalizer_fn=None) * FLAGS.text_scale
the FLAGS.text_scale is 512 default
but in eval.py, I can't  find where to change it, since the size in eval.py is random (multiple of 32).
if it is still 512, is there some errors since the input size isn't 512?
thanks
